spec/views/pages/attend.html.erb_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "pages/attend.html.erb",  type: :view do
+
+    it "Should render what you will learn section" do
+      render "shared/what_you_will_learn.html.erb"
```

Comment:
dont render another page than the one you're describing, see comment in #54 or #45 about it